regimes/pl/extensions.go

```diff
@@ -34,34 +34,43 @@ var extensionKeys = []*cbc.KeyDefinition{
 		},
 	},
 	{
-		Key: ExtKeyKSeFVATZero,
+		Key: ExtKeyKSeFVATRegion,
 		Name: i18n.String{
-			i18n.EN: "Zero VAT Extensions for KSeF",
+			i18n.EN: "Region VAT Extensions for KSeF",
 		},
-		Keys: []*cbc.KeyDefinition{
+		Codes: []*cbc.CodeDefinition{
 			{
-				Key: "wdt",
+				Code: "EU",
 				Name: i18n.String{
-					i18n.EN: "WDT",
-					i18n.PL: "WDT",
+					i18n.EN: "EU",
+					i18n.PL: "EU",
+				},
+				Desc: i18n.String{
+					i18n.EN: "Inter european trade",
+					i18n.PL: "Handel wewnątrzwspólnotowy.",
```

Comment:
It seems to me that the word "trade" is used less often in this context and it is more about "transactions". I was inspired by this [Polish website](https://www.biznes.gov.pl/en/publikacje/2274-transakcje-wewnatrzwspolnotowe).

```suggestion
					i18n.EN: "Intra-EU Transactions",
					i18n.PL: "Transakcje wewnątrzwspólnotowe",
```

I would wonder if this is a change in the right direction. I mean is the expansion of the shortcut:
> WDT - Wewnątrzwspólnotowa dostawa towarów

Maybe we should keep WDT and use the above description?

Reply:
I don't like the idea of keeping WDT because it is Polish specyfic shortcut and we would prefer to keep things more international. It is also specific for transporting goods where I want to make it more generic.